ptl_tests/utils.py

```diff
@@ -0,0 +1,171 @@
+import pytorch_lightning as pl
+from torch import nn
+import torch
```

Comment:
let's make sure we don't put random files at the top level directory? :) put it into somewhere in train/ or air/

Reply:
Oh I forget to clean them. I'll put all these tests files into ray/train folder.